lib/utils/file.js

```diff
@@ -0,0 +1,17 @@
+// Based on https://github.com/kevva/executable
+// Since this module doesn't expose the function to check stat mode only,
+// his logic is pasted here.
+// Opened an issue and a pull request about it,
+// to maybe switch to module in the future:
+// Issue: https://github.com/kevva/executable/issues/9
+// PR: https://github.com/kevva/executable/pull/10
+
+export function isExecutable (fileStat) {
+  if (process.platform === 'win32') return true;
```

Comment:
`cd <drop in file/folder location>` and adding a return is a nice feature 🎉 

Yes, thats my $0.05, but I won't stop you 👻

Maybe the magic is an awesome feature a lot of people will like, but if you ask me, I think it should be a plugin 📌 😊

I don't often drop files to the terminal, but when i do it's mostly stuff from chrome like images to use with `dd` 👼